tests/hpc/rasdaemon.pm

```diff
@@ -0,0 +1,40 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: rasdaemon may have to be tested on a physical system, as EDAC support
+#may not be available in a VM. To check this, install rasdaemon:
+# zypper in rasdaemon and run
+# ras-mc-ctl --status.
+#If the message returned indicates that there is no EDAC driver loaded, the system  does
+# not support it thus not all tests below can be performed
+# Maintainer: Anton Smorodskyi <asmorodskyi@suse.com>
+# Tags: https://fate.suse.com/318824
+
+use base 'hpcbase';
+use strict;
+use testapi;
+use utils;
+
+sub run {
+    my $self = shift;
+
+    # Install genders package
+    zypper_call('in rasdaemon');
+
+    script_run('ras-mc-ctl --status');
+    systemctl 'start rasdaemon';
+    my $summary_output = script_output('ras-mc-ctl --summary');
+    die 'Not expected summary - ' . $summary_output
+      unless $summary_output =~ /No Memory errors/ && $summary_output =~ /No PCIe AER errors/ && $summary_output =~ /No MCE errors/;
+    my $error_output = script_output('ras-mc-ctl --error');
+    die 'Not expected error - ' . $error_output
+      unless $error_output =~ /No Memory errors/ && $error_output =~ /No PCIe AER errors/ && $error_output =~ /No MCE errors/;
+}
```

Comment:
@asmorodskyi Could you please add comments to this part? Just short description of above actions.

I also see in fate `ras-mc-ctl --mainboard` and `ras-mc-ctl --layout` did you try them?
Those commands have  output with some value/information:
```
phoenix:~ # ras-mc-ctl --mainboard
ras-mc-ctl: mainboard: ASUSTeK COMPUTER INC. model MAXIMUS VIII HERO
phoenix:~ # ras-mc-ctl --layout
ras-mc-ctl: Error: No memories found at via edac.
```

Reply:
At least mainboard work in vm:
```
ras-mc-ctl --mainboard
ras-mc-ctl: mainboard: QEMU model Standard PC (i440FX + PIIX, 1996)
```

Reply:
@asmorodskyi you are right, it is not mandatory to start rasdaemon prior ras-mc-ctl --status